blocks/toc/toc.css

```diff
@@ -1,3 +1,22 @@
 .toc-list-empty {
   display: none;
+}
+
+ul.toc-list {
+  list-style-type: none;
+  padding-inline: 0;
+}
+  
+ul.toc-sub-list {
+  list-style-type: none;
+  padding-inline: 24px;
+}
+  
+li.toc-h2 {
+  font-weight: 700;
+}
+  
+li.toc-h3 {
+  margin: 16px 0;
+  font-weight: 500;
```

Comment:
@sforschen, in a separate ticket we might want to consider abstracting the font-weights that we use in a css variable so we have an idea of which weights we use and which ones we don't recommend using.

Reply:
Absolutely, I have already found some variables in the code already. I will add a separate issue about unifying and updating these.